meta-tpm/recipes-tpm/trousers/files/trousers.init.sh

```diff
@@ -27,7 +27,7 @@ case "${1}" in
 	start)
 		echo "Starting $DESC: "
 
-		if [ ! -e /dev/tpm* ]
+		if ! ls /dev/tpm* >/dev/null 2>/dev/null
```

Comment:
Please change to `if ! ls /dev/tpm* >/dev/null 2>&1`

Reply:
Makes sense, done.